Modules/Core/Transform/include/itkTransform.h

```diff
@@ -115,6 +115,14 @@ class ITK_TEMPLATE_EXPORT Transform : public TransformBaseTemplate<TParametersVa
     return VOutputDimension;
   }
 
+  /* For storing the  name of InputSpace */
+  itkSetMacro(InputSpaceName, std::string);
+  itkGetConstReferenceMacro(InputSpaceName, std::string);
+
+  /* For storing the  name of OutputSpace */
+  itkSetMacro(OutputSpaceName, std::string);
+  itkGetConstReferenceMacro(OutputSpaceName, std::string);
```

Comment:
Please exercise the added methods in the corresponding C++ test.

Reply:
Done in 5da379a. Thanks @PranjalSahu and @thewtex for the comment to ensure that the calls return what is expected.